src/sql/parts/dashboard/dashboard.module.ts

```diff
@@ -128,7 +129,7 @@ const appRoutes: Routes = [
 	providers: [
 		{ provide: APP_BASE_HREF, useValue: '/' },
 		{ provide: IBreadcrumbService, useClass: BreadcrumbService },
-		DashboardServiceInterface,
+		{ provide: CommonServiceInterface, useClass: DashboardServiceInterface },
```

Comment:
This is fine, but I think you should also define 
```
{ provide: DashboardServiceInterface, useClass: DashboardServiceInterface },
```
and allow classes to depend on DashboardServiceInterface for things that require it. As mentioned in a previous comment we should probably depend on CommonServiceInterface wherever possible so that users can use components wherever it makes sense, but some things will need dashboard-specific functions